app/poros/suggestion_facade.rb

```diff
@@ -0,0 +1,10 @@
+class SuggestionFacade
+	def initialize(current_user)
+		@current_user = current_user
+	end
+
+	def schedule_activities
+		scheduler = SchedulerService.new(@current_user)
```

Comment:
Good abstraction here pulling our service into ScheduleService from the Suggestions Facade